Automattic/wp-calypso
# Remove confusing help tooltip for free plan users

Removes the following tooltip:

![screenshot-2018-04-24-17-05-21](https://user-images.githubusercontent.com/205419/39355409-7384afe4-4a0d-11e8-9d21-68aceca6b72b.png)

It does not communicate anything useful

Test plan:
* Go to plans page as a free user
* Confirm the tooltip is not there
* Go to plans page as a paid user
* Confirm the tooltip is there and says `We'll deduct the cost of your current plan from the full price (%(price)s) for the next 12 months`
